app/components/autocomplete/slash_suggestion/mentions/index.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {
+    autocompleteChannels,
+    AutocompleteSuggestion,
+    autocompleteUsersInChannel,
+    Channel,
+    COMMAND_SUGGESTION_CHANNEL,
+    COMMAND_SUGGESTION_USER,
+    DispatchFunc,
+    GlobalState,
+    UserAutocomplete,
+    UserProfile,
+} from '../app_command_parser/app_command_parser_dependencies';
+
+interface Store {
+    dispatch: DispatchFunc;
+    getState: () => GlobalState;
+}
+
+export async function inTextMentionSuggestions(pretext: string, store: Store, channelID: string, teamID: string, delimiter = ''): Promise<AutocompleteSuggestion[] | null> {
```

Comment:
this functions looks like utils or helpers, can we move them out from the components folder?